antlr-kotlin-runtime/src/commonTest/kotlin/MiniCalcParserTest.kt

```diff
@@ -53,6 +54,31 @@ class MiniCalcParserTest : BaseTest() {
         assertEquals("\n", newline.text)
     }
 
+    @test
+    fun testOperatorPrecedence() {
+        val input = CharStreams.fromString("print(-foo + bar)\n")
+        val lexer = MiniCalcLexer(input)
+        val parser = MiniCalcParser(CommonTokenStream(lexer))
+
+        val root = parser.miniCalcFile()
+
+        val line = checkNotNull(root.findLine(0))
+
+        val statement = line.findStatement() as MiniCalcParser.PrintStatementContext
+
+        val print = checkNotNull(statement.findPrint())
+
+        val plus = print.findExpression() as MiniCalcParser.BinaryOperationContext
+        checkNotNull(plus.PLUS())
```

Comment:
The test fails here because the expression is interpreted to be `-(foo + bar)` instead of `(-foo) + bar`.